packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -161,15 +164,17 @@ const mapConfigurations = {
             // TODO doesn't work atm; no coordinate source
             geometry: false,
             window: true,
-            properties: {
-              filename: 'Name of file',
-            },
+            properties: { filename: 'Name of file' },
           },
         },
         coordinateSources: [], // to be set in addPlugins.ts
       },
       pins: commonPins,
       reverseGeocoder,
+      meldemichelAfmButton: {
```

Comment:
Why not just `afmButton`?

Otherwise I'd suggest adding an overall directive `meldemichel` in which `afmButton` is a parameter - this is probably not the last local plugin we are writing.

Reply:
Sounds good. https://github.com/Dataport/polar/pull/23/commits/7199419d10018cb5dfef40b80006fe450bf6e451